spynnaker/pyNN/models/neural_projections/connectors/abstract_connector_supports_views_on_machine.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2017-2019 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+class AbstractConnectorSupportsViewsOnMachine(object):
```

Comment:
It might be easier to have some sort of "size_in_words" method in here instead of repeating the constants in each of the subclasses.